_variables.yml

```diff
@@ -20,5 +8,29 @@ design:
   crud: |
     An acronym for Create, Read, Update, and Delete. These four actions are the
     core operations for data storage on computers.
+  data-resource: |
+    The data layer of the framework and its contents. 
+    A data resource will stores data from a research study or data source.
```

Comment:
```suggestion
    Is the research data and metadata, and any management needed, of a specific research study. 
```

Reply:
Only *one* study? Couldn't it be multiple studies?

Reply:
This is where it gets a bit tricky with the language. Right now, a "Data Resource" corresponds to `project id` in Sprout. So you can have multiple "projects" within Sprout that are "Data Resources". In Garden/Propagate, there *might* also be `projects` that correspond to analyses using the Data Resource (aka `projects` in Sprout). But I'm not sure how exactly that will look like yet, the exact naming and structure details haven't been done yet 😛

Reply:
Even the word "study" is a bit confusing. When writing a paper we would say "in this study we did this and that"... but the study is actually an analysis of a larger "study". We researchers don't usually say "this is a substudy of..." unless there are studies described within a specific paper.

Reply:
Ah, I see. I agree that the word "study" can be confusing in this context. 
I actually thought that a Data Resource corresponded somewhat to a database, so that one Data Resource could  contain multiple projects?

Reply:
@signekb, hmm, now we get into what a project means 😛 A project for a single paper? Or a project for a single database/set of data files? If the first, yes you are right. If the second, than no 😛